### ISSUE TITLE
Should hero banner be marked as "decorative"

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov_base#419]
This came up during testing with a client:

> Can the homepage hero be decorative/skipped alt attribute?
>
> Due to the nature of the banner image against the page this should never hold value to understanding context of page, therefore should be skipped so it's not distracting.

What are our thougths?

**Comment by willguv** [comment]
Yes, think so. It's not essential to the understanding of the page and announcing the alt tag delays the user reaching the main nav

**Comment by msayoung** [comment]
Sorry @markconroy - I didnt see this earlier. 

I disagree that it should be ignored by default. 
It completely depends on the image that has been used - in which case marking it decorative should be on a case by case basis.

An approach would be to add a decorative image flag to the  banner image paragraph. 
See: https://github.com/localgovdrupal/localgov/issues/506

**Comment by finnlewis** [comment]
@msayoung and @markconroy could you confirm the preferred approach for https://github.com/localgovdrupal/localgov_base/pull/420 ?

**Comment by markconroy** [comment]
I'm happy to agree with Maria, and we fix this on a case-by-case basis for each client. Or file a follow up issue if we need it to add a flag to the paragraph type.

**Comment by markconroy** [comment]
After 1.5 years, we have had no dissenting voices to treat the banner as a 'case-by-case' item, so I'm going to close this.
